deploy/charts/csi-driver-spiffe/Chart.yaml

```diff
@@ -13,4 +13,4 @@ sources:
 - https://github.com/cert-manager/csi-driver-spiffe
 
 appVersion: v0.2.0
-version: v0.2.2
+version: v0.2.3
```

Comment:
suggestion: I'd rather not change the version in this PR, if possible. I'd rather bump it when we do a new release, in a separate PR!